app/main.py

```diff
@@ -1,8 +1,58 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self, comfort_class: int,
+        clean_mark: int,
+        brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: float,
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark >= self.clean_power:
+            return 0.0
+
+        price = (
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating
+            / self.distance_from_city_center
+        )
+        return round(price, 1)
+
+    def wash_single_car(self, car: Car) -> bool:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+            return True
+        return False
```

Comment:
This violates the requirement: "CarWashStation.wash_single_car(car: Car) must: ... This method does not return a value (or returns None)." Currently the method is defined as `def wash_single_car(self, car: Car) -> bool:` and returns `True`/`False`. Remove the boolean returns and change the signature to `-> None`; simply update `car.clean_mark` when `self.clean_power > car.clean_mark` and do not return any value. See the task description for the exact requirement and examples .

Reply:
Karol, this method should not return any value, but currently it's returning a boolean. Besides that, this method is correct - just remove the return statements and change the signature to -> None.

```suggestion
    def wash_single_car(self, car: Car) -> None:
        if car.clean_mark < self.clean_power:
            car.clean_mark = self.clean_power
```